src/generic/Debug.cpp

```diff
@@ -64,8 +64,8 @@ void Debug::registerKeywords( Keywords& keys ) {
   Action::registerKeywords( keys );
   ActionPilot::registerKeywords(keys);
   keys.add("compulsory","STRIDE","1","the frequency with which this action is to be performed");
-  keys.addFlag("logActivity",false,"write in the log which actions are inactive and which are inactive");
-  keys.addFlag("logRequestedAtoms",false,"write in the log which atoms have been requested at a given time");
+  keys.addFlag("LOGACTIVITY",false,"write in the log which actions are inactive and which are inactive");
```

Comment:
@Iximiel here, for consistency with previous versions, it might be wiser to keep the current capitalization and then automatically make upper case all flags. I am saying this in case people use the new manual prepare an input for an old version. But perhaps it is irrelevant (we switched to case insensitive quite some time ago, in v2.6).